sugondat-chain/runtimes/sugondat-kusama/tests/integration_test.rs

```diff
@@ -1,3 +1,4 @@
+use crate::fee_adjustment::*;
```

Comment:
Glob imports are bad style and should be avoided except through specifically crafted preludes.

Reply:
Maybe, as an exception, you could do it in the `use super::*` in an inline mod, but I am not a fan tbh.

Reply:
To be fair, I really don't know why I used a global import. However, `fee_adjustment::{BlobsLengthToFee, NextLengthMultiplier, TransactionByteFee}` are the necessary imports, so it's not a big deal 😅 I have updated the code, and there are no more global imports